### PR TITLE
BAH-512 | Maharjun | removed disposition types whose having two empty va…

### PR DESCRIPTION
**Pull request by maharjunm**
… page or patient image

https://bahmni.atlassian.net/browse/BAH-512

### CODE REVIEW

**Comment by angshu** [comment]
Please reflect the commit message with the JIRA card number.

File: ui/app/clinical/consultation/controllers/dispositionController.js [hunk 1]
```diff
@@ -32,9 +32,11 @@ angular.module('bahmni.clinical')
             });
         };
 
-        var findAction = function (dispositions, action) {
-            var undoDischarge = _.find(dispositions, action);
-            return undoDischarge || {'name': ''};
+        var pushDispositionAction = function (finalDispositionActions, dispositions, action) {
+            var dispositionPresent = _.find(dispositions, action);
+            if (dispositionPresent) {
+                finalDispositionActions.push(dispositionPresent);
```
Comment (sumanmaity112) [inline ui/app/clinical/consultation/controllers/dispositionController.js]: Hi @maharjunm,
**pushDispositionAction** method should not modify the parameter (**finalDispositionActions**). It should return some value

File: ui/app/clinical/consultation/controllers/dispositionController.js [hunk 2]
```diff
@@ -32,9 +32,13 @@ angular.module('bahmni.clinical')
...
 
-        var findAction = function (dispositions, action) {
-            var undoDischarge = _.find(dispositions, action);
-            return undoDischarge || {'name': ''};
+        var pushDispositionAction = function (finalDispositionActions, dispositions, action) {
+            var copyOfFinalDispositionActions = _.cloneDeep(finalDispositionActions);
+            var dispositionPresent = _.find(dispositions, action);
+            if (dispositionPresent) {
+                copyOfFinalDispositionActions.push(dispositionPresent);
+                return copyOfFinalDispositionActions;
```
Comment (sumanmaity112) [inline ui/app/clinical/consultation/controllers/dispositionController.js]: Hi @maharjunm ,
I think this `return` will be outside of `if` block. As if **dispositionPresent** is false, ** finalDispositionActions** will be `undefined`

File: ui/test/unit/clinical/controllers/dispositionController.spec.js [hunk 3]
```diff
@@ -106,10 +106,10 @@ describe("DispositionController", function () {
         expect(scope.dispositionActions).toContain({"name":"extraDisposition","code":""});
     });
 
-    it("should give empty value if any of the default values are deleted", function () {
+    iit("should give undefined value if any of the default values are deleted", function () {
```
Comment (sumanmaity112) [inline ui/test/unit/clinical/controllers/dispositionController.spec.js]: Hi @maharjunm,
Remove `iit`

**Comment by sai789** [comment]
@sumanmaity112 We made changes according to your feedback. Please have a look at it.

**Pull request merged**